test/tstCompileOnlyAccessTraits.cpp

```diff
@@ -84,4 +82,8 @@ void test_access_traits_compile_only()
   // check_valid_access_traits(PrimitivesTag{}, InvalidMemorySpace{});
 
   // check_valid_access_traits(PrimitivesTag{}, SizeMemberFunctionNotStatic{});
+
+  // check_valid_access_traits(ArborX::Traits::PrimitivesTag{}, p);
```

Comment:
This one does not give an error message

Reply:
That's what I was saying, but you [said](https://github.com/arborx/ArborX/pull/667#discussion_r838948054) that you got the warning

Reply:
My point was this is under `// Uncomment to see error messages`
The statement give warnings not an error.
I would just remove it.